src/service/comment.js

```diff
@@ -107,14 +107,16 @@ const createChildCommentV3 = async (
       message: text,
       sendPostNotif,
       activityId,
-      anonimity: isAnonymous
+      anonimity: isAnonymous,
+      is_you: true
     };
     const anonimity = {
       emoji_name: anonUser.emojiName,
       color_name: anonUser.colorName,
       emoji_code: anonUser.emojiCode,
       color_code: anonUser.colorCode,
-      is_anonymous: isAnonymous
+      is_anonymous: isAnonymous,
+      is_you: true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the `is_you` property is added to the `anonimity` object and is always set to `true`. If this function is used to create anonymous comments for other users, this might not be correct. Please ensure that this property is correctly set based on the user context.

```diff
-      is_anonymous: isAnonymous,
-      is_you: true
+      is_anonymous: isAnonymous,
+      is_you: currentUser === commentAuthor
```
Please replace `currentUser` and `commentAuthor` with the appropriate variables in your codebase that represent the current user and the author of the comment respectively.


<!-- This is an auto-generated comment by OSS CodeRabbit -->